electroncash_plugins/fusion/plugin.py

```diff
@@ -598,6 +601,21 @@ def run(self, ):
                     for f in list(wallet._fusions_auto):
                         f.stop('Wallet has unconfirmed coins... waiting.', not_if_running = True)
                     continue
+
+                fuse_depth = Conf(wallet).fuse_depth
+                if fuse_depth > 0:
+                    sum_eligible_values = 0
+                    sum_fuz_values = 0
+                    for (eaddr, ecoins) in eligible:
+                        ecoins_value = 0
+                        for ecoin in ecoins:
+                            ecoins_value += ecoin['value']
+                        sum_eligible_values += ecoins_value
+                        if self.is_fuz_address(wallet, eaddr, fuse_depth - 1):
+                            sum_fuz_values += ecoins_value
+                    if sum_eligible_values != 0 and float(sum_fuz_values / sum_eligible_values) >= FUSE_DEPTH_THRESHOLD:
```

Comment:
Nit - all `/` division in python already produces floats.. the `float()` cast is superfluous

```suggestion
                    if sum_eligible_values != 0 and sum_fuz_values / sum_eligible_values >= FUSE_DEPTH_THRESHOLD:
```